docs/operate-a-stake-pool/cardano-relay-configuration.md

```diff
@@ -111,13 +120,13 @@ PORT=6000
 # 0.0.0.0 listens on all local IP addresses for the computer
 HOSTADDR=0.0.0.0
 # Set a variable to indicate the file path to your topology file
-TOPOLOGY=$HOME/cardano-testnet/topology.json
+TOPOLOGY=<$HOME>/cardano-testnet/topology.json
```

Comment:
The point of the angle brackets is to force the user to substitute the `$HOME` variable, but has the side effect of bombing out the script with an erroneous file redirect if the user doesn't substitute that string as requested.

At least in the prior version of the code there is a change `$HOME` will be defined correctly.

If such a change needs to be made I would recommend an undefined literal like `insertHomeDirectoryHere` which catches the user's attention and would also be present in a meaningful error message if the user doesn't update the directory.

Reply:
@sanskys this problem will go away if we proceed as tentatively agreed in https://github.com/cardano-foundation/developer-portal/pull/1162#issuecomment-1787925058.